.github/workflows/build-vcpkg.yml

```diff
@@ -166,3 +164,38 @@ jobs:
         shell: "bash"
         run: |
           cmake --build . ${{ matrix.cmake_build_options }}
+
+  build-docker:
+    strategy:
+      matrix:
+        os: ["centos-7"]
+      fail-fast: false
+
+    name: "Build with vcpkg"
+    needs: check-skip
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout PR
+        if: ${{ needs.check-skip.outputs.platform }}
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        if: ${{ needs.check-skip.outputs.platform }}
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          driver: docker
+
+      - name: CentOS-7 image
```

Comment:
minor: as this is a matrix of OS's flavour (which happens to only have 1 element in it at the moment), this step should really have a generic name.